parts/linux/cloud-init/artifacts/aks-check-network.sh

```diff
@@ -34,7 +34,7 @@ declare -A URL_LIST=(
     ["eastus.data.mcr.microsoft.com"]="FQDN *.data.mcr.microsoft.com is required for MCR storage backed by the Azure content delivery network (CDN)."\
     ["login.microsoftonline.com"]="This is equired for Microsoft Entra authentication."\
     ["packages.microsoft.com"]="This is required to download packages (like Moby, PowerShell, and Azure CLI) using cached apt-get operations."\
-    ["acs-mirror.azureedge.net"]="This is required to download and install required binaries like kubenet and Azure CNI."\
+    ["acs-mirror.azureedge.net/acs-mirror/healthz"]="This checks connection to CDN which is required to download and install required binaries like kubenet and Azure CNI."\
```

Comment:
do we want to use --head ?  I see that hte curl command isn't

Reply:
Oh for this one I don't think it's as important - this is only triggered by hand, meant for troubleshooting purposes. I could add this, but on NPD side it's more important I think

Reply:
Added --head when allowed